pkg/sys/types.go

```diff
@@ -20,6 +20,10 @@ type BlockDevice struct {
 	Devname     string      `json:"devName,omitempty"`
 	Devtype     string      `json:"devType,omitempty"`
 	Partitions  []Partition `json:"partitions,omitempty"`
+	DMName      string      `json:"dmName,omitempty"`
```

Comment:
Embed a new struct instead of repeating the same set of fields twice

```
type DMDevice {
	DMName    string `json:"dmName,omitempty"`	
	DMUUID    string `json:"dmUUID,omitempty"`	
	Master    string `json:"master,omitempty"`	
	Parent    string `json:"parent,omitempty"`
}

type BlockDevice struct {
       ....
       DMDevice
}

type Partition struct {
        ....
        DMDevice
}
```

Reply:
Change the structure name as `MasterInfo` instead of `DMDevice` for later `MDUUID` addition if supported.